stubs/s2clientprotocol/s2clientprotocol/raw_pb2.pyi

```diff
@@ -92,9 +92,7 @@ global___CloakState = CloakState
 
 @typing.final
 class StartRaw(google.protobuf.message.Message):
-    """
-    Start
```

Comment:
Might be better to leave this alone and keep the docstrings exactly as they are in the source; that makes them easier to validate. Don't feel too strongly though.